pages/feed/index.tsx

```diff
@@ -116,6 +116,9 @@ const Feeds: WithPageLayout<HighlightSSRProps> = (props: HighlightSSRProps) => {
           image={props?.ogImage}
           twitterCard="summary_large_image"
         />
+        <div className="hidden">
```

Comment:
Glad this works... But I'm still not sure how 😂😂

Reply:
😂  They say it in the docs but its not that clear. If the `/feed` page was completely server-rendered, we wouldn't need the hidden form.

https://docs.netlify.com/forms/setup/#work-with-javascript-rendered-forms